version.py

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env python3
+
+# This script finds and prints the various versions in this project: wasi-sdk
+# itself, LLVM, and the Git revisions of dependencies.
+#
+# Usage: version [wasi-sdk|llvm|llvm-major|dump] [--llvm-dir=<non-project dir>]
+
+import sys
+import argparse
+import subprocess
+
+# The number of characters to use for the abbreviated Git revision.
+GIT_REF_LEN = 12
+
+
+def exec(command, cwd=None):
+    result = subprocess.run(command, capture_output=True, text=True, cwd=cwd)
```

Comment:
You can use `stdout=PIPE` instead I think

Reply:
In fact `stdout=PIPE` is better in this case I think since `capture_output` will redirect (and potentially hide) stderr message which could be useful for debugging.

Reply:
Good suggestion, though I ended up just adding some error-handling code here anyways that captures `stderr`.